lua/telescope/builtin/files.lua

```diff
@@ -210,6 +217,10 @@ files.tags = function(opts)
     sorter = conf.generic_sorter(opts),
     attach_mappings = function()
       actions._goto_file_selection:enhance {
+        pre = function()
+          local entry = actions.get_selected_entry()
+          entry.filename = entry.resolve_filename(entry.filename)
```

Comment:
Not sure if there was a non-mutate-the-world way of doing this or if this was what you were suggesting but yeah let me know.

>

Reply:
That looks good. Does it work as well?

Reply:
Indeed.

Reply:
(I also don't 100% recall how method syntax works in lua, I'm sure there's a way to write that as `entry:resolve_filename` right? but my attention span is quickly waning since there are 10 other things I need to fix after this, so yeah this was as good as I came up with for now)

Reply:
No it looks good. `:` is used if we actually do something like a class. E.g. `lua/telescope/pickers.lua`. This just assign's a function pointer to a table entry. Not sure if it is called function pointer but thats basically it.